x-pack/plugins/actions/server/saved_objects/migrations.ts

```diff
@@ -62,22 +65,36 @@ export function getMigrations(
     pipeMigrations(addisMissingSecretsField)
   );
 
+  const migrationResolveSavedObjectsIdsInActionTaskParams = createEsoMigration(
```

Comment:
With [this PR](https://github.com/elastic/kibana/pull/108964), we will be storing the `actionId` for `action_task_params` in the `references` array, so all SOs of this type should be migrated. I don't think we need this check for `doc.attributes.hasOwnProperty('relatedSavedObjects')`